warehouse/warehouse/scripts/validate_cmip.py

```diff
@@ -102,10 +102,12 @@ def main():
     p.add_argument('output', help="Path to where generated plots should be saved")
     p.add_argument('dataset_id', help="The dataset_id of the variable being validated")
     args = p.parse_args()
-    plot_seasonal_decomp(args.input, args.dataset_id, args.output)
+
+    # elided plotting due to unfathomable errors in imports
```

Comment:
Yeah, this one should be fix-able, eventually.  I'll add.
 However, it should NOT trigger failure to produce the cmorized output (from temp dirs to the real-deal) due to a graphing failure.  Viewing the graphics is a human-in-the-loop issue.  As it stands, a graphics failure would require regenerate of the data - totally unnecessary.

Reply:
Yup I agree, that step should not bottleneck the entire workflow.

Maybe adding a flag like `--debug` and a conditional to ignore certain steps (e.g., this one) would help, instead of removing it altogether (unless it really isn't needed)?

Reply:
Some inline programmatic checks are good - we should implement more.  The publisher (esgpublish) employs "PrePARE" and Autocurator, for metadata sanity.  But to determine if the data variable values are "sane/reasonable", Sterling implemented an inline generation of images.  I suppose once might catch "ambient temperature drops to absolute zero" this way, or other cataclysmic data errors.  But not sure the wisdom.  In any case, such generation should be on a side track. (Cool if we could pull them up using a web-based data management dashboard, along with dataset status views, publications outstanding, etc. - future work).

Reply:
Thanks for the clarification. From what it sounds like, it seems like the image generation was more eyeballing as a quality assurance check.